nix/sources.json

```diff
@@ -1,26 +1,26 @@
 {
     "haskell.nix": {
-        "branch": "master",
+        "branch": "jbgi/cardano-wallet-fix-windows",
```

Comment:
This needs to be master branch, so your PRs on Haskell.nix need to be merged first.

Reply:
agreed, though cardano-wallet master is currently not using haskell.nix master but https://github.com/input-output-hk/haskell.nix/tree/hkm/cardano-wallet-fix

Reply:
cardano-wallet master is on haskell.nix master I'm pretty sure.

Reply:
It is not, see https://app.slack.com/client/T0N639Z4N/GDB7LERDY/thread/CG386Q1NE-1582215136.357100 for context. (maybe the issue has been solved on master since then, cc/ @hamishmack)